app/controllers/dashboard_controller.rb

```diff
@@ -1,6 +1,9 @@
 class DashboardController < ApplicationController
   def show
-    if !current_user
+    if current_user
+      @user = current_user
+      @invitations = Invitation.where("receiver_id = ? AND accepted IS NULL", @user.id)
```

Comment:
Don't need to use `@user` here. Can use `current_user` in query for invitations and in the view (as it is a helper method)

Reply:
could use AR relationships with invitations for user (sent / received)

Reply:
Fixed. The query on line 5 has been moved to the `user` model, accessed through `user.outstanding_invitations`.